src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -2904,6 +2911,11 @@
                 <ref name="classes"/>
             </list>
         </attribute>
+        <optional>
+            <attribute name="aria-describedby">
+                <data type="string"/>
+            </attribute>
+        </optional>
```

Comment:
This puts the `aria-describedby` attribute on the `<figure>` element. It should be on the `<img>` element. Since it is optional in the RelaxNG, it could be put into the existing definition of the `<img>` element.

Reply:
See also discussion on #435: https://github.com/nlbdev/nordic-epub3-dtbook-migrator/pull/435#pullrequestreview-615321336

Maybe we don't need to declare it here at all.